pkg/service/init.go

```diff
@@ -87,20 +81,22 @@ func (b *Bootstrap) BootstrapHandler(ctx context.Context, wg *sync.WaitGroup, st
 		},
 	})
 
-	err = provision.LoadProfiles(configuration.Device.ProfilesDir, dic)
+	ds.controller.InitRestRoutes(dic)
```

Comment:
`InitRestRoutes` also was moved much later in the sequence. As before, was there a reason for this change?

Reply:
Because device command REST API frequently check the cache to ensure that device, deviceResource exist. So I moved the `InitRestRoutes` after `InitCache`, but before `LoadProfiles` and `LoadDevices` so that callback can be successfully triggered.

Reply:
OK, thanks for the explanation.